synapse/models/infotech.py

```diff
@@ -956,6 +966,28 @@ def getModelDefs(self):
                         'doc': 'The most recent version of the rule evaluated as a match.'}),
                 )),
 
+                ('it:reveng:function', {}, ()),
```

Comment:
this almost feels like it could use a name + desc secondary property pair; if only to allow people to encode what/why something is so special.

Reply:
I like that idea :+1: